nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -518,6 +518,10 @@ In addition to numerous new and upgraded packages, this release has the followin
   you should change the package you refer to. If you don't need them update your
   commands from `otelcontribcol` to `otelcorecol` and enjoy a 7x smaller binary.
 
+- `services.zookeeper` has a new option `jre` for specifying the JRE to start
+  zookeeper with. It defaults to the JRE built with `pkgs.zookeeper` instead of
+  `pkgs.jre`.
```

Comment:
Thanks!

And I'm also curious on why not just set `ExecStart=${pkgs.zookeeper}/bin/zkServer.sh start-foreground` or simply wrap around `zkServer.sh` 😃

Reply:
I don't know. Maybe it can be done and it would probably be an improvement.
If someone relied on a specific (lack of) behavior, I assume they would have added a comment or a test assertion.

Reply:
I don't use zookeeper heavily and probably can't test it well.

Maybe I can have a try along with version bump (latest ver is 3.8.0) in the future. 😀

Thanks for reply!